### ISSUE TITLE
Version 3.0.1: ImportError: No module named fcntl

### ISSUE THREAD

**Issue by erm0l0v** [WoLpH/python-progressbar#27]
Hi I got trouble with installing new version (3.0.1) on Windows:

In my case progressbar2 installed as a dependency of *django-stdimage*.

Console output:
```
Collecting progressbar2>=2.7 (from django-stdimage==1.2.2->-r requirements\base.txt (line 22))
  Using cached progressbar2-3.0.1.tar.gz
    Complete output from command python setup.py egg_info:
    Traceback (most recent call last):
      File "<string>", line 20, in <module>
      File "c:\users\jenssd~1\appdata\local\temp\pip-build-rnf6vf\progressbar2\setup.py", line 6, in <module>
        from progressbar import metadata
      File "progressbar\__init__.py", line 64, in <module>
        from .bar import ProgressBar
      File "progressbar\bar.py", line 5, in <module>
        import fcntl
    ImportError: No module named fcntl
    
    ----------------------------------------
Command "python setup.py egg_info" failed with error code 1 in c:\users\jenssd~1\appdata\local\temp\pip-build-rnf6vf\progressbar2
```

**Comment by WoLpH** [comment]
Hmm, that's a serious issue. Apparently fcntl is not supported by Windows (hard to test for me unfortunately), I'll get out a fix today :)

**Comment by WoLpH** [comment]
This should be fixed by the new release (3.1.0), please give it a try and let me know if you have any issues :)

**Comment by erm0l0v** [comment]
Yes this version works fine. Thank you!
